cortipy/cortical_client.py

```diff
@@ -153,16 +154,16 @@ def _queryAPI(self, method, resourcePath, queryParams,
       response = requests.post(
         url, params=queryParams, headers=headers, data=postData)
     else:
-      raise Exception('Method ' + method + ' is not recognized.')
+      raise RequestMethodError('Method ' + method + ' is not recognized.')
     if response.status_code != 200:
-      raise Exception("Response " + str(response.status_code)
-                      + ": " + response.content)
+      raise UnsuccessfulEncodingError("Response " + str(response.status_code)
+                               + ": " + response.content)
```

Comment:
fix indentation. And when line breaking on an operator it is more common to leave the operator on the previous line so it is obvious that the statement continues to the next line